app/Http/Controllers/MovieController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use App\Models\Review;
+use Illuminate\Http\Request;
+
+class MovieController extends Controller
+{
+    public function index()
+    {
+        $movies = Movie::latest()->get();
+        return view('movies.index', ['movies' => $movies]);
+    }
+
+    public function create()
+    {
+        //
+    }
+
+    public function store(Request $request)
+    {
+        //
+    }
+
+    public function show(Movie $movie)
+    {
+        $movie->with(['genres', 'celebs', 'reviews'])->get();
+        $duration = explode('.', (string) round($movie->duration/60, 2));
```

Comment:
My friend have you tried using number_format($movie->duration/60, 2, '.')? 
Also, if the duration is used in another instance, perhaps you can add this as an attribute?

In Laravel, you can format attributes and add them to the model [see docs](https://laravel.com/docs/8.x/eloquent-mutators)

You could have a method `getDurationAttribute`, then have that format there. That way every time you do `$movie->duration` this will already be formatted. If you think you would use the duration without format for something, you could name the method like `getDurationForHumansAttribute`, then to access it, you just do it like the following $movie->duration_for_humans 😸

Reply:
My friend! I've made the changes but unsure if they right! Please let me know :)